src/array.js

```diff
@@ -1,66 +1,66 @@
 if (typeof Uint8Array !== "undefined") {
-  var crossfilter_array8 = function(n) { return new Uint8Array(n); };
-  var crossfilter_array16 = function(n) { return new Uint16Array(n); };
-  var crossfilter_array32 = function(n) { return new Uint32Array(n); };
+  var array8 = function(n) { return new Uint8Array(n); };
+  var array16 = function(n) { return new Uint16Array(n); };
+  var array32 = function(n) { return new Uint32Array(n); };
 
-  var crossfilter_arrayLengthen = function(array, length) {
+  var arrayLengthen = function(array, length) {
     if (array.length >= length) return array;
     var copy = new array.constructor(length);
     copy.set(array);
     return copy;
   };
 
-  var crossfilter_arrayWiden = function(array, width) {
+  var arrayWiden = function(array, width) {
     var copy;
     switch (width) {
-      case 16: copy = crossfilter_array16(array.length); break;
-      case 32: copy = crossfilter_array32(array.length); break;
+      case 16: copy = array16(array.length); break;
+      case 32: copy = array32(array.length); break;
       default: throw new Error("invalid array width!");
     }
     copy.set(array);
     return copy;
   };
 }
 
-function crossfilter_arrayUntyped(n) {
+function arrayUntyped(n) {
   var array = new Array(n), i = -1;
   while (++i < n) array[i] = 0;
   return array;
 }
 
-function crossfilter_arrayLengthenUntyped(array, length) {
+function arrayLengthenUntyped(array, length) {
   var n = array.length;
   while (n < length) array[n++] = 0;
   return array;
 }
 
-function crossfilter_arrayWidenUntyped(array, width) {
+function arrayWidenUntyped(array, width) {
   if (width > 32) throw new Error("invalid array width!");
   return array;
 }
 
 // An arbitrarily-wide array of bitmasks
-function crossfilter_bitarray(n) {
+function bitarray(n) {
```

Comment:
I have to admit that I'm not a javascript expert, but can't this be changed to an ES6 class?